pkg/api/admin_provisioning.go

```diff
@@ -21,6 +22,14 @@ func (server *HTTPServer) AdminProvisioningReloadDatasources(c *models.ReqContex
 	return Success("Datasources config reloaded")
 }
 
+func (server *HTTPServer) AdminProvisioningReloadApps(c *models.ReqContext) Response {
+	err := server.ProvisioningService.ProvisionApps()
+	if err != nil {
+		return Error(500, "", err)
```

Comment:
I'm not sure it's safe to return the error message to the client? For the sake of security, I think we should log the error here:

```suggestion
		log.Error("Provisioning apps failed", "error", err)
		return Error(500, "", "Internal server error")
```

Reply:
No, it's not needed. Happening automatically, see https://github.com/grafana/grafana/blob/7b7000c5bcb445dcf039d50ab4feed68842ef056/pkg/api/common.go#L91 which uses message "Internal Server Error" and https://github.com/grafana/grafana/blob/7b7000c5bcb445dcf039d50ab4feed68842ef056/pkg/api/common.go#L49-L52 which log the error.

Reply:
Ah OK, a bit unexpected from the API.